lib/rules/no-bare-strings.js

```diff
@@ -113,8 +91,8 @@ const IGNORED_ELEMENTS = new Set(['pre', 'script', 'style', 'template', 'textare
 // Character entity reference chart: https://dev.w3.org/html5/html-author/charref
 const DEFAULT_CONFIG = {
   allowlist: DEFAULT_ALLOWLIST,
-  globalAttributes: GLOBAL_ATTRIBUTES,
-  elementAttributes: TAG_ATTRIBUTES,
+  globalAttributes: DEFAULT_GLOBAL_ATTRIBUTES,
+  elementAttributes: DEFAULT_ELEMENT_ATTRIBUTES,
```

Comment:
maybe something like:
```suggestion
  elementAttributes: DEFAULT_ATTRIBUTES_BY_ELEMENT,
```

Reply:
I agree that's a better name, but I want to be consistent with the public option name for now.